microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -50,7 +50,7 @@ Follow the interactive tutorial sections in this article to learn the basics of
 
 1. In the `aloha` class, find the value for the `padding` style and copy it.
 
-   ![CSS class applied to the inspected element is highlighted in the Styles panel](./index-images/css-elements-inspect-me-styles.png)
+   ![CSS class applied to the inspected element is highlighted in the Styles panel](./index-images/inspect-me-styles.png)
 
 1. On the page view, paste the `padding` value in the **Value of padding:** text box.
```

Comment:
```suggestion
1. On the page view, paste the `padding` value in the **Value of padding** text box.
```
* per style guide, for UI label "Foo...", just write **Foo**, not **Foo...**.  Similar reasoning for colon in UI label.